.github/workflows/quickstart_ci.yml

```diff
@@ -52,6 +66,16 @@ jobs:
           java-version: ${{ matrix.jdk }}
           distribution: 'temurin'
           cache: 'maven'
+      - name: Run before script
+        env:
+          FILE: "./quickstarts/.github/workflows/quickstart_${{ inputs.QUICKSTART_PATH }}_ci_before.sh"
```

Comment:
Maybe but this one and the other file as the env for the whole job to reduce duplication? IMO having it at the top level also makes it more 'documented'

Reply:
My idea was to have enforce each QS stays independent .yml and .sh wise. Duplication should be rare, if even will exist, and I will of course document this is at the google doc.